Tests/TestingTests/PollingTests.swift

```diff
@@ -0,0 +1,556 @@
+      }
+      #expect(issues.count == 1)
+    }
+
+    @available(_clockAPI, *)
+    @Test("if the closure continues to pass")
+    func continuousCalling() async throws {
+      let incrementor = Incrementor()
+
+      try await confirmation(until: stop) {
+        _ = await incrementor.increment()
+        return true
+      }
+
+      #expect(await incrementor.count > 1)
+    }
+
+    @available(_clockAPI, *)
+    @Test("Thrown errors will automatically exit & fail")
+    func errorsReported() async {
+      let issues = await runTest {
+        try await confirmation(until: stop) {
+          throw PollingTestSampleError.ohNo
+        }
+      }
+      #expect(issues.count == 1)
+    }
+
+    @available(_clockAPI, *)
+    @Test("Calculates how many times to poll based on the duration & interval")
+    func defaultPollingCount() async throws {
+      let incrementor = Incrementor()
+      try await confirmation(until: stop, pollingEvery: .milliseconds(1)) {
+        await incrementor.increment() != 0
+      }
+      #expect(await incrementor.count == 1000)
+    }
+
+    @Suite(
+      "Configuration traits",
+      .pollingConfirmationDefaults(
+        until: .stopsPassing,
+        within: .milliseconds(100)
+      )
+    )
+    struct WithConfigurationTraits {
+      let stop = PollingStopCondition.stopsPassing
+
+      @available(_clockAPI, *)
+      @Test(
+        "When no test/callsite configuration, it uses the suite configuration"
+      )
+      func testUsesSuiteConfiguration() async throws {
+        let incrementor = Incrementor()
+        try await confirmation(until: stop, pollingEvery: .milliseconds(1)) {
+          await incrementor.increment() != 0
+        }
+        let count = await incrementor.count
+        #expect(count == 100)
+      }
+
+      @available(_clockAPI, *)
+      @Test(
+        "Ignore trait configurations that don't match the stop condition",
+        .pollingConfirmationDefaults(
+          until: .firstPass,
+          within: .milliseconds(
+            500
+          )
+        )
+      ) func testIgnoresTraitsWithNonmatchingStopConditions() async throws {
+        let incrementor = Incrementor()
+        try await confirmation(until: stop, pollingEvery: .milliseconds(1)) {
+          await incrementor.increment() != 0
+        }
+        let count = await incrementor.count
+        #expect(count == 100)
+      }
+
+      @available(_clockAPI, *)
+      @Test(
+        "When test configuration provided, uses the test configuration",
+        .pollingConfirmationDefaults(
+          until: .stopsPassing,
+          within: .milliseconds(10)
+        )
+      )
+      func testUsesTestConfigurationOverSuiteConfiguration() async throws  {
+        let incrementor = Incrementor()
+        try await confirmation(until: stop, pollingEvery: .milliseconds(1)) {
+          await incrementor.increment() != 0
+        }
+        let count = await incrementor.count
+        #expect(await count == 10)
```

Comment:
I think this `await` is unnecessary:
```suggestion
        #expect(count == 10)
```